src/OrbitGl/TimeGraph.h

```diff
@@ -44,7 +44,7 @@ class TimeGraph : public orbit_gl::CaptureViewElement {
   ~TimeGraph() override;
 
   void Draw(Batcher& batcher, TextRenderer& text_renderer, uint64_t current_mouse_time_ns,
-            PickingMode picking_mode = PickingMode::kNone, float /*z_offset*/ = 0) override;
+            PickingMode picking_mode, uint32_t = 0, float = 0) override;
```

Comment:
Nit: `uint32_t /*indentation_level*/ = 0, float /*z_offset*/ = 0`

Reply:
It's an old commit issue, I will fix it when rebasing.